marklogic-data-hub-central/ui/src/config/bench.config.ts

```diff
@@ -216,6 +216,106 @@ const jobRespSuccess = {
   "status": 200
 };
 
+const flowsJSON = {
+  "data": [{
+    "name": "testFlow",
+    "description": "",
+    "batchSize": 100,
+    "threadCount": 4,
+    "stopOnError": false,
+    "options": {},
+    "version": 0,
+    "steps": {
+      "1": {
+        "name": "loadJSON",
+        "description": "",
+        "options": {
+          "sourceQuery": null,
+          "collections": ["loadJSON"],
+          "loadData": {
+            "name": "loadJSON"
+          },
+          "outputFormat": "json",
```

Comment:
@wooldridge can you pls with latest develop if a flow artifact stores output format. Mine doesnt seem to.

Reply:
```
"1": {
  "name": "loadCustomer", 
  "options": {
    "loadData": {
      "name": "loadCustomer"
    }
  }, 
  "stepDefinitionName": "default-ingestion", 
  "stepDefinitionType": "INGESTION"
}
```

Reply:
@bsrikan is right. It is in settings doc not in the flow

Reply:
Also these details might change depending how loadData is rewritten. To keep it simple, maybe we can just test with an xml load data now to keep refactoring to a minimum in the future. Thoughts?

Reply:
Yes, I think this can be revisited once loadData / settings is rewritten